pkg/pipelines/bootstrap.go

```diff
@@ -141,6 +145,7 @@ func bootstrapResources(o *BootstrapOptions, appFs afero.Fs) (res.Resources, err
 	}
 	secretFilename := filepath.Join("03-secrets", secretName+".yaml")
 	secretsPath := filepath.Join(config.PathForPipelines(cfg), "base", secretFilename)
+	logBootstrapStatus(fmt.Sprintf("Webhook secret to authenticate pull/push to %s repo created sucessfully", serviceName))
```

Comment:
We could use `log.Infof` for formatting instead of `fmt.Sprintf`

Reply:
Will make the changes:)

Reply:
The changes have been addressed